orcoursetrion/lib/github.py

```diff
@@ -64,6 +66,38 @@ def _get_all(self, url):
                 results += response.json()
         return results
 
+    def _find_team(self, org, team):
+        """Find a team in an org by name, or raise.
+
+        Args:
+            org (str): Organization to create the repo in.
+            team (str): Team to find by name.
+
+        Raises:
+            GitHubUnknownError
+            GitHubNoTeamFound
+
+        Returns:
+            dict: Team dictionary
+                  (https://developer.github.com/v3/orgs/teams/#response)
+        """
+        list_teams_url = '{url}orgs/{org}/teams'.format(
+            url=self.api_url,
+            org=org
+        )
+        teams = self._get_all(list_teams_url)
+        if not teams:
+            raise GitHubUnknownError(
+                'No teams found in {0} organization'.format(org)
+            )
+        found_team = [x for x in teams if x['name'].strip() == team.strip()]
```

Comment:
I've found for most devs list comprehensions are easier to grok than lambda closures, but I do like yours better. I also realized I should probably case fix while I'm at it